### PR TITLE
Create HAProxy_Configuration.md

### PR DESCRIPTION
**Pull request by LeslieLundquist**

### CODE REVIEW

File: _neutron/HAProxy_Configuration.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+weight: 2
+tags: [neutron, haproxy, configuration]
+author: Peter Xiang Wang, Sangeetha Srikanth
+dateAdded: June 1st, 2016
+editor: Leslie Lundquist
+---
+
+## HAProxy Configuration for Large Number of Networks
+
+When you connect to a large number of Neutron networks, the default HAProxy buffer size will not be large enough for the Neutron CLI to list such a large number. You will see the following error message:
```
Comment (ssrikant20) [inline _neutron/HAProxy_Configuration.md:14]: When customer provisions a large number of Neutron networks,

**Comment by nirajdp76** [comment]
LGTM

File: _neutron/HAProxy_Configuration.md (hunk 1 above)
Comment (LeslieLundquist) [inline _neutron/HAProxy_Configuration.md:14]: @ssrikant20 Sangeetha, is this document intended for customers or operations staff?

**Comment by nirajdp76** [comment]
How will this be categorized as knowledge base article for Ops. It is something that ops can use when they see customer provisioning 100s of network in neutron and hit this problem.

**Comment by LeslieLundquist** [comment]
@nirajdp76 @ssrikant20 OK, I misunderstood and I will move this article into the Ops repository.

**Comment by xwang0818** [comment]
The buffer size is default 16384 in main.yml. It needs to be increased to 32768 for this fix.

**Comment by LeslieLundquist** [comment]
Closed because this article belongs in the Ops repo. Opening a new PR there.

**Pull request closed without merging**